Exercises/kokkoskernels/BlockJacobi/Begin/blockjacobi.cpp

```diff
@@ -12,12 +12,9 @@
 #include "KokkosBatched_Copy_Impl.hpp"
 #include "KokkosBatched_SetIdentity_Decl.hpp"
 #include "KokkosBatched_SetIdentity_Impl.hpp"
-#include "KokkosBatched_Gemv_Decl.hpp"
-#include "KokkosBatched_Gemv_Team_Impl.hpp"
+#include "KokkosBlas2_team_gemv.hpp"
 #include "KokkosBatched_Trsm_Decl.hpp"
-#include "KokkosBatched_Trsm_Team_Impl.hpp"
 #include "KokkosBatched_LU_Decl.hpp"
-#include "KokkosBatched_LU_Team_Impl.hpp"
```

Comment:
Is that really all the headers we expect users to include?

Reply:
Yes, users are only supposed to include the Decl files (each one includes the corresponding Impl file)